src/Solnet.Wallet/Utilities/Base58Encoder.cs

```diff
@@ -170,5 +172,23 @@ public override byte[] DecodeData(string encoded)
                 vch[i2++] = b256[it2++];
             return vch;
         }
+
+
+        /// <summary>
+        /// Strict validation with no whitespace allowed
+        /// </summary>
+        /// <param name="value">Base58 string data</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static bool IsValidWithoutWhitespace(string value)
+        {

```

Comment:
Please delete the method `IsMaybeEncoded ` that had same functionality but is now obsolete given yours should beat it in performance by several magnitudes due to lookup dictionary.

Reply:
Done